apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.IntegrationTests/ErrorReporting/ErrorReportingTest.cs

```diff
@@ -64,8 +64,7 @@ public async Task LogsException()
 
             Assert.Equal(HttpStatusCode.InternalServerError, response.StatusCode);
 
-            var errorEvent = s_polling.GetEvents(_testId, 1).Single();
-
+            var errorEvent = ErrorEventEntryVerifiers.VerifySingle(s_polling.GetEvents(_testId, 1), _testId);
```

Comment:
Could this be simplified to `ErrorEventEntryVerifiers.VerifySingle(s_polling, _testId)`? We always call `GetEvents(testId, 1)` for `VerifySingle`, and we always pass in the same testId to VerifySingle as we do to GetEvents.

Reply:
I left it as is in case we could reuse VerifySingle and VerifyMany for collections obtained in different ways, but it's true that we don't need that right now, we can always make it more general if we need to. Will submit a commit shortly.